### ISSUE TITLE
New repository owner

### ISSUE THREAD

**Issue by gdgib** [vlebourl/custom_vesync#182]
@vlebourl I figured it might be useful to start a discussion here.

I'll admit I definitely don't have time to really maintain this either, but with 9 devices I'm also not exactly going to stop using it.  If we can't find someone with the proper time to add features, frankly I'll likely end up at least semi-maintaining it out of self-interest.

I'm an expert in open source, programming, embedded systems, and basically everything else necessary.  If it weren't for a shortage of time, I'd be an ideal candidate.

Bottom line: if there's no one better, I'd be pleased to at least keep things going for the forseable future.

**Comment by gdgib** [comment]
You might remember I started a fork with a couple of minor fixes a while back.  I do remember you asked me to clean things up somewhat, it just hasn't gotten to the top of my list of smarthome projects.  Sigh...

**Comment by vlebourl** [comment]
Hey, I do remember, and I thank you for that, and your offer. As issues keep being opened despite the header in the Readme, I was on the  verge of archiving the repo. Would you rather have me archive the repo and redirect to your fork or transfer ownership?

**Comment by vlebourl** [comment]
I added you as a collab for now, that could be a start

**Comment by wheimbigner** [comment]
For what it's worth - and for some reason I think I made a comment like this a while back but I can't find it/don't remember so apologies if this is a repeat:

Under the hood, the "smart" Levoit Humidifiers and purifiers are [controls that use spicy wiring]<-->[??? comm protocol]<-->[MCU that handles touch controls etc.]<-->[simple UART protocol]<-->ESP32 chip. ESP32 handles all the wifi, bluetooth, "smart" functionality. On the Levoit LV600S humidifier, there's practically a cutout where an esp32 chip was stapled-in after-the-fact to an existing board to add smart functionality.

Anyways, these chips are ESPHome-able (if you solder in a serial terminal... it's also very likely possible via enabling developer mode and doing something OTA with firmware, but that would be out of my league for now, and at any rate, if you enable developer mode, you also have LAN control). I've started building a custom component to play nice with the Core 200S, and I've no doubt that basically all of the other VeSync equipment works in a similar manner.

Anyways, for me, reverse engineering the UART protocol etc. was the easy/fun part, it's the coding part I'm horrible at.

I realize there's only so much overlap between the "I want my VeSync devices accessible from Home Assistant" crowd vs the "I want to totally cloud-cut my Vesync device and put custom firmware on it" crowd, but to whatever extent I can help, I'm happy to do so.

(NB: In the firmware, there's very limited info on the API between phone app and vesync servers - the devices interact with vesync over MQTT)

**Comment by slonob** [comment]
I'd like to collaborate as well.  I started an implementation of one Levoit VeSync purifier and humidifier model in SmartThings but realized that I had no interest in adapting to the changes Samsung was making.  I'll work on what I need, which is LUH-A602S-WUS (LV600S) device support.

**Comment by vlebourl** [comment]
One of you should fork the project and make it the new reference.
I'll archive this repo in the meantime. I guess people can still use this component while archived.
Sorry I can't do more...
